CycloneDX/Program.cs

```diff
@@ -128,9 +127,8 @@ class Program {
 #endregion options
 
         internal static IFileSystem fileSystem = new FileSystem();
-        internal static readonly IJsonDocs jsonDoc = new JsonDocs();
         internal static readonly IDotnetCommandService dotnetCommandService = new DotnetCommandService();
-        internal static readonly IProjectAssetsFileService projectAssetsFileService = new ProjectAssetsFileService(fileSystem, dotnetCommandService, () => new AssetFileReader(), jsonDoc);
```

Comment:
This is reverted to the 2.7.0 state, as this change is not needed by my approach.